classification/signals/classification_search.py

```diff
@@ -18,16 +18,19 @@
     search_type=Classification,
     example=SearchExample(
         note="The lab record ID",
-        examples=["vc1545"]
+        examples=["CR_1545"]
```

Comment:
Can you make this example dependant on the setting? I think things should be in an order where you could say

examples=["CR_1545" if settings.VARIANT_CLASSIFICATION_ID_OVERRIDE_PREFIX else "vc1545"]